src/a-st-ext.adoc

```diff
@@ -505,7 +505,7 @@ All atomic operations follow the rules in xref:section_cheri_int_load_store_insn
 where the atomic operation is considered to be a load followed by a store for the
 capability rules.
 
-Therefore _all_ atomic operations are _reserved_ if `cs1=c0`.
+Therefore _all_ atomic operations are _reserved_ if `cs1=y0`.
```

Comment:
```suggestion
Therefore _all_ atomic operations are _reserved_ if `rs1=y0`.
```

I'm expecting to change these to rs1/rs2/rd as normal risc-v - to be confirmed on tuesday

Reply:
yeah - let's keep cs1 until the tuesday meeting